packages/mobile/src/components/TokenDisplay.tsx

```diff
@@ -91,7 +93,7 @@ function TokenDisplay({
 
   const amountToShow = showLocalAmount ? amountInLocalCurrency : new BigNumber(amount)
 
-  const sign = amountToShow.isNegative() ? '-' : showExplicitPositiveSign ? '+' : ''
+  const sign = hideSign ? '' : amountToShow.isNegative() ? '-' : showExplicitPositiveSign ? '+' : ''
```

Comment:
Why do we need to hide the sign?

Reply:
The Figma mockups say that. The amount in local currency has the + or - but the token amount doesn't have any sign: https://www.figma.com/file/8USELhisKuP1YlCt95WrD2/Support-for-Multiple-Tokens?node-id=951%3A29262